packages/@aws-cdk/aws-s3-deployment/README.md

```diff
@@ -59,10 +59,40 @@ all but a single file:
 
 ## Retain on Delete
 
-By default, the contents of the destination bucket will be deleted when the
+By default, the contents of the destination bucket will **not** be deleted when the
```

Comment:
I changed this because apparently the default behavior was changed in this [PR](https://github.com/aws/aws-cdk/pull/1006/files), but the README was not updated.